test/extensions/bootstrap/wasm/test_data/logging_rust.rs

```diff
@@ -6,44 +6,45 @@ use proxy_wasm;
 #[no_mangle]
 pub fn proxy_abi_version_0_1_0() {}
 
-use log::{debug, error, info, trace, warn};
+use log::*;
 
 extern "C" {
-    fn proxy_done() -> u32;
+  fn proxy_done() -> u32;
 }
 
 #[no_mangle]
-pub fn proxy_on_context_create(_context_id: u32, _parent_context_id_: u32) {}
-
-#[no_mangle]
-pub fn proxy_on_configure(_context_id: u32, _plugin_configuration_size: u32) -> u32 {
-    trace!("test trace logging");
-    debug!("test debug logging");
-    error!("test error logging");
+pub fn proxy_on_configure(_context_id : u32, _plugin_configuration_size: u32) -> u32 {
     warn!("warn configure-test");
     1
 }
 
 #[no_mangle]
-pub fn proxy_on_vm_start(_context_id: u32, _vm_configuration_size: u32) -> u32 {
+pub fn proxy_on_vm_start(_context_id : u32, _vm_configuration_size : u32) -> u32 {
+    trace!("test trace logging");
+    debug!("test debug logging");
+    error!("test error logging");
     1
 }
 
 #[no_mangle]
-pub fn proxy_on_tick(_context_id: u32) {
+pub fn proxy_on_tick(_context_id : u32) {
     info!("test tick logging");
     unsafe {
         proxy_done();
     }
 }
 
 #[no_mangle]
-pub fn proxy_on_done(_context_id: u32) -> u32 {
+pub fn proxy_on_done(_context_id : u32) -> u32 {
     info!("onDone logging");
     0
 }
 
 #[no_mangle]
-pub fn proxy_on_delete(_context_id: u32) {
+pub fn proxy_on_delete(_context_id : u32) {
     info!("onDelete logging");
 }
+
+// This function is to compile WebAssembly code with `rust_binary` with Bazel.
+// Envoy wasm runtime won't take care of this type of function.
+fn main() {}
```

Comment:
Sorry, I don't understand why do we need this? Doesn't `wasm32` target take care of it?

Also, no new line at end of file.